tf/cloudrun.tf

```diff
@@ -0,0 +1,85 @@
+
+resource "google_cloud_run_service" "bwdserver" {
+  name     = "bwdserver"
+  location = "us-west1"
+
+  template {
+
+    metadata {
+      annotations = {
+        "autoscaling.knative.dev/minScale" : "0"
+        "autoscaling.knative.dev/maxScale" : "10"
+        "run.googleapis.com/cloudsql-instances" : "balmy-ground-195100:us-west1:dark-west"
+        "run.googleapis.com/startup-cpu-boost" : "true"
+        "run.googleapis.com/cpu-throttling" : "true"
+        "run.googleapis.com/execution-environment" : "gen2"
+      }
+    }
+
+    spec {
+      timeout_seconds      = 300
+      service_account_name = "cloud-run-runner@balmy-ground-195100.iam.gserviceaccount.com"
+      containers {
+        image = "gcr.io/balmy-ground-195100/gcp-fsharp-bwdserver@sha256:ec7bbcd9e38d8965b76d5718359fac1242fefdcb11ae7250ed918415ecd829a3"
```

Comment:
I haven't worked out the deploy strategy. I think it might actually be to leave this blank in terraform and merge via specifically updating this field via google apis.